### ISSUE TITLE
What Does Categorization regex Actually Match Against?

### ISSUE THREAD

**Issue by Anti-Distinctlyminty** [ActivityWatch/activitywatch#523]
tl;dr What is the categorization regular expression matching against? What is the regex flavour? Needs documentation.

I have searched the documentation concerning the categorization, and although I'm almost certain that the categorization is done against the active window title (in the case of the aw-watcher-window), it is not actually stated anywhere what the categorization regex is matching against, or what the flavour of regex is. 

The reason I ask is because I am trying to use this regex 
`(.+?\.hip(?:lc|nc)*)\s-\sHoudini\s.*`
to be able to match something like
`//192.168.x.xxx/Some/Path/SomeFile.hip - Houdini FX 18.0.566`
Which I tested on regex101.com and works fine, but fails to apply the category in ActivityWatch.

Some additional documentation, in the docs and/or the settings window,  would be welcome.

**Comment by ErikBjare** [comment]
The regex matches against window title and app name (and in the future also the URL).

The regex flavor for aw-server(-python) is whatever Python's `re` module uses. Might be slightly different in aw-server-rust.

However, we've had some issues with escaping which will be fixed in the next release (only the first escaped character works properly in v0.10).

I agree that improved docs/instructions in the settings panel would be nice. I'd gladly accept a PR for it.

**Comment by johan-bjareholt** [comment]
I made a PR on the docs explaining categorization, rules and regex in the docs

https://github.com/ActivityWatch/docs/pull/52

**Comment by avinashkanaujiya** [comment]
> The regex matches against window title and app name (and in the future also the URL).

Is the URL implemented?

**Comment by BelKed** [comment]
No
